helix-core/src/indent.rs

```diff
@@ -40,20 +43,13 @@ impl IndentStyle {
     pub fn as_str(&self) -> &'static str {
         match *self {
             IndentStyle::Tabs => "\t",
-            IndentStyle::Spaces(1) => " ",
-            IndentStyle::Spaces(2) => "  ",
-            IndentStyle::Spaces(3) => "   ",
-            IndentStyle::Spaces(4) => "    ",
-            IndentStyle::Spaces(5) => "     ",
-            IndentStyle::Spaces(6) => "      ",
-            IndentStyle::Spaces(7) => "       ",
-            IndentStyle::Spaces(8) => "        ",
-
+            IndentStyle::Spaces(n) if n <= 16 && n >= 1 => &INDENTS[0..n as usize],
```

Comment:
The last two branches can be merged by just unconditionally clamping. That fixes the clippy libt too that makes CI fails